RFCs/005-Framework-Extensibility-Custom-DataSource.md

```diff
@@ -89,3 +89,6 @@ The MSTest v2 framework, on discovering a `TestMethod` probes additional attribu
 1. Users can extend `ITestDataSource` to support custom data sources.
 2. Multiple tests can reuse the test data defined in same data source.
 3. A test case can use multiple test data sources.
+
+### Remarks
+When implementing a custom `ITestDataSource` (attribute), the `GetData()` method should not return an empty sequence, otherwise the test(s) using this data source attribute will always fail.
```

Comment:
Can you elaborate on this please. Will tests fail along with any specific diagnostic? Such elaboration might help the reader; hence my comment.

Reply:
@pvlakshm The test fails with following message:
Error in executing test. No result returned by extension. If using extension of TestMethodAttribute then please contact vendor.

Reply:
On second thoughts, perhaps not.